include/zephyr/bluetooth/mesh/access.h

```diff
@@ -187,6 +187,10 @@ struct bt_mesh_elem {
 #define BT_MESH_MODEL_ID_REMOTE_PROV_SRV           0x0004
 /** Remote Provisioning Client */
 #define BT_MESH_MODEL_ID_REMOTE_PROV_CLI           0x0005
+/** Bridge Configuration Sever */
```

Comment:
```suggestion
/** Subnetwork Bridge Configuration Sever */
```

Reply:
In the Mesh Protocol specification it is named "Bridge Configuration Server". I would prefer to keep the names as close as possible to what is in the specification if it not another good reason not do so. In the specification it also called Bridge Configuration Client, Bridging Table, Bridging Table Size. Subnet Bridge is only the state if the functionality is enabled or disabled.

Reply:
Ok, agree. Then just ignore my comments related to this. However, the comment about the common prefix is still valid.

Reply:
I think the common prefix should be brg as short for Bridge. I think that is the best prefix. I checked in allacronyms page and that is used for "Postal, Technology, Telecom", so I think that is the best prefix for bridge.